### ISSUE TITLE
Exception when creating a VRF interface

### ISSUE THREAD

**Issue by tmmorin** [svinota/pyroute2#416]
CommitException: target vrf_table is not set  is raised by this kind of piece of code:

```
        with ipdb.create(kind='vrf',
                            ifname=self.vrf_if,
                            vrf_table=self.rt_table,
                            reuse=True) as i:
            i.up()
```

```
  File "/opt/stack/new/networking-bagpipe/networking_bagpipe/bagpipe_bgp/vpn/ipvpn/mpls_linux_dataplane.py", line 91, in __init__
    i.up()
  File "/opt/stack/new/networking-bagpipe/.tox/dsvm-fullstack/local/lib/python2.7/site-packages/pyroute2/ipdb/transactional.py", line 209, in __exit__
    self.commit()
  File "/opt/stack/new/networking-bagpipe/.tox/dsvm-fullstack/local/lib/python2.7/site-packages/pyroute2/ipdb/interfaces.py", line 1043, in commit
    raise error
CommitException: target vrf_table is not set
```

(this hits 0.4.20 but was ok in 0.4.17)

**Comment by svinota** [comment]
Strange, 'cause nothing has changed in VRF code since 0.4.17.

Ok, let me see.

**Comment by svinota** [comment]
I confirm the bug. Thanks a lot!

Fixing now.

**Comment by svinota** [comment]
The root couse found, will be fixed tonight.

**Comment by svinota** [comment]
Hotfix released. It works in the sense it worked in 0.4.17, but the root cause **not** fixed properly — so will be fixed in the next releases. Not closing the issue yet.

**Comment by tmmorin** [comment]
Thanks!
(testing in progress...)

**Comment by tmmorin** [comment]
0.4.21 fixes this regression, thanks!

**Comment by tmmorin** [comment]
Ah sorry, I closed the issue while you wanted to keep it open...

**Comment by svinota** [comment]
Should be fixed for releases starting from 0.5.0
